src/cloudflare/internal/http.ts

```diff
@@ -1,4 +1,6 @@
 export interface Fetcher {
-  fetch: typeof fetch;
+  // We don't use typeof fetch here because that would use the client-side signature,
```

Comment:
Wait, this type is actually supposed to be the client-side type. `Fetcher` is the type of a service binding.